### DISCUSSION TITLE
Parcel 2 Missing Babel Runtime

### DISCUSSION THREAD

**Discussion by fab7jj** [parcel-bundler/parcel#4727] [General] [answered]
Hello I went to install the twilio-chat yarn package and upon running parcel watch I got the below error

![babel runtime](https://user-images.githubusercontent.com/40164436/84391798-bdef3b80-abf9-11ea-86e8-8e70f640690e.PNG)

I had to install this [babel runtime yarn package](https://yarnpkg.com/package/babel-runtime) for it to work. First I tried [this one](https://yarnpkg.com/package/@babel/runtime) but it still didn't work for some reason. To be honest I don't know what is the difference between the two versions it is a bit confusing.

Is this a bug of parcel or it is normal that you have to install that package as a separate dependency? I know people who installed the twilio-chat package while using create-react-app and hadn't this issue for example.

Thanks a lot.

**Comment by mischnic** [comment, marked as answer]
> it is normal that you have to install that package as a separate dependency

No, this is an error on twilio's side. 
They import `babel-runtime` in their package (see https://unpkg.com/browse/twilio-chat@3.4.0/browser/client.js line 3) but didn't declare it as a dependency in their package.json: https://unpkg.com/browse/twilio-chat@3.4.0/package.json

> I know people who installed the twilio-chat package while using create-react-app and hadn't this issue for example.

This happened because some other package coincidentally had `babel-runtime` as a dependency and so it was already available. But this is prone to break in the future

> To be honest I don't know what is the difference between the two versions it is a bit confusing.

When `babel-runtime` version 7 was released, they changed the package name to `@babel/runtime`. But you have no choice here (as you already found out).

**Reply by fab7jj** [reply]
Thanks mate, really appreciate you taking your time and explaining this to me. Great work!
